js/src/Ice/OutgoingResponse.js

```diff
@@ -8,7 +8,7 @@ export class OutgoingResponse {
     constructor(outputStream, replyStatus, exceptionId, exceptionDetails) {
         this._outputStream = outputStream;
         this._replyStatus = replyStatus || ReplyStatus.Ok;
-        if (replyStatus !== ReplyStatus.Ok) {
+        if (replyStatus !== ReplyStatus.Ok.value) {
```

Comment:
The `.value` looks suspicious. The `replayStatus` is a enumerator not a byte.

Reply:
You're right, will fix.